packages/react/src/components/Chat/ChatItem.tsx

```diff
@@ -89,23 +90,62 @@ class ChatItem extends UIComponent<ReactProps<ChatItemProps>, any> {
   }
 
   private renderChatItem(styles: ComponentSlotStylesPrepared) {
-    const { message, gutter, contentPosition } = this.props
+    const { gutter, contentPosition } = this.props
     const gutterElement =
       gutter &&
       Box.create(gutter, {
         defaultProps: { className: ChatItem.slotClassNames.gutter, styles: styles.gutter },
       })
 
+    const messageElement = this.getMessageAugmentedWithChatMessageProps(styles)
+
     return (
       <>
         {contentPosition === 'start' && gutterElement}
-        {Box.create(message, {
-          defaultProps: { className: ChatItem.slotClassNames.message, styles: styles.message },
-        })}
+        {messageElement}
         {contentPosition === 'end' && gutterElement}
       </>
     )
   }
+
+  getMessageAugmentedWithChatMessageProps = styles => {
+    const { message, attached } = this.props
+    const messageElement = Box.create(message, {
+      defaultProps: {
+        className: ChatItem.slotClassNames.message,
+        styles: styles.message,
+      },
+    })
+
+    // the box element is ChatMessage
```

Comment:
comment may be a bit misleading - what we are testing here is not the `Box` element, but rather if messageElement is ChatMessage. Lets just omit `box` from the comment text